GCRCatalogs/catalog_configs/base5000_v1.1.1.yaml

```diff
@@ -2,8 +2,8 @@
 
 subclass_name: cosmodc2.BaseDC2GalaxyCatalog
 
-catalog_root_dir: /global/cfs/cdirs/lsst/catalogs/baseDC2_5000_v1.1.1
-catalog_filename_template: baseDC2_z_{}_{}_cutout_{}.hdf5
+catalog_root_dir: /global/cfs/cdirs/lsst/shared/xgal/skysim/base5000_v1.1.1
```

Comment:
In the new scheme this path needs to be
^/xgal/skysim/base5000_v1.1.1

Reply:
I just pushed this change. Should be ready to merge now.